snappy/common_test.go

```diff
@@ -106,25 +106,26 @@ apps:
 		return "", err
 	}
 
-	if err := storeMinimalRemoteManifest(m.Name, testDeveloper, m.Version, "Hello...", "remote-channel"); err != nil {
+	if err := storeMinimalRemoteManifest(m.Name, testDeveloper, m.Version, "hello in summary", "Hello...", "remote-channel"); err != nil {
 		return "", err
 	}
 
 	return yamlFile, nil
 }
 
-func storeMinimalRemoteManifest(name, developer, version, desc, channel string) error {
+func storeMinimalRemoteManifest(name, developer, version, summary, desc, channel string) error {
 	if developer == SideloadedDeveloper {
 		panic("store remote manifest for sideloaded package")
 	}
-	info := &snap.Info{
-		Name:        name,
-		Version:     version,
-		Developer:   developer,
-		Description: desc,
-		Channel:     channel,
+	sideInfo := snap.SideInfo{
+		OfficialName: name,
+		//Version:           version,
```

Comment:
Would it make sense to remove this commented out line? It seems like we do not really need `Version` in SideInfo (revision yes).